cmd/validator/slashing-protection/export.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/prysmaticlabs/prysm/io/file"
 	"github.com/prysmaticlabs/prysm/validator/accounts/userprompt"
 	"github.com/prysmaticlabs/prysm/validator/db/kv"
-	export "github.com/prysmaticlabs/prysm/validator/slashing-protection/local/standard-protection-format"
+	slashingprotection "github.com/prysmaticlabs/prysm/validator/slashing-protection-history"
 	"github.com/urfave/cli/v2"
 )
 
 const (
 	jsonExportFileName = "slashing_protection.json"
 )
 
-// ExportSlashingProtectionJSONCli extracts a validator's slashing protection
+// Extracts a validator's slashing protection
```

Comment:
The doc should begin with the function's name.

Reply:
Not if the function is unexported